docs/setup-dev-environment.md

```diff
@@ -1,43 +1,116 @@
+
+For "Colouring London", we have found that the size of the database means that a VM with access to 50GB of storage is appropriate. If you are using the OSM test data, the default storage settings in VirtualBox should suffice.
+
+##### In either case, you should set the memory to `2048` MB.
+
+If you a running Ubuntu in a virtual environment you will need to configure networking to forward ports from the guest to the host. For Virtual Box the following was configured under NAT port forwarding (found under `Settings -> Network -> Advanced -> Port Forwarding`).
+
+Name     | Protocol  | Host Port  | Guest Port
+-------- | --------- | ---------- | -----------
+app      | TCP       | 8080       | 3000
+app_dev  | TCP       | 3001       | 3001
+ssh      | TCP       | 4022       | 22
+
+The `app_dev` mapping is used in development by Razzle which rebuilds and serves client side assets on the fly.
+
+To run the commands in the rest of this setup guide, either `ssh` into the VirtualBox environment or open the terminal within the Ubuntu GUI.
+
+If you wish to `ssh`, you will first need to open the terminal in Ubuntu and run the following.
+
+```bash
+sudo apt-get install -y openssh-server
+```
+
+You can then `ssh` into the VirtualBox VM set up with the port  forwarding described above like so, where `<linuxusername>` is the name you set up during the installation of Ubuntu (you can type `whoami` in the Ubuntu terminal to remind yourself of this).
+
+```bash
+ssh <linuxusername>@localhost -p 4022
+```
+</details>
+
+## Contents
+
+- [:tulip: Installing the tools and components](#tulip-installing-the-tools-and-components)
+  - [:red_circle: Installing PostgreSQL](#red_circle-installing-postgresql)
+  - [:rainbow: Installing Colouring London](#rainbow-installing-colouring-london)
+  - [:arrow_down: Installing Node.js](#arrow_down-installing-nodejs)
+  - [:large_blue_circle: Configuring PostgreSQL](#large_blue_circle-configuring-postgresql)
+  - [:arrow_forward: Configuring Node.js](#arrow_forward-configuring-nodejs)
+- [:house: Loading the building data](#house-loading-the-building-data)
+- [:computer: Running the application](#computer-running-the-application)
+  - [:eyes: Viewing the application](#eyes-viewing-the-application)
+
+## :tulip: Installing the tools and components
 
 First upgrade the installed packages to the latest versions, to remove any security warnings.
 
+```bash
+sudo apt-get update -y
+sudo apt-get upgrade -y
 ```
-sudo apt-get update
-sudo apt-get upgrade
+
+Now install some essential tools.
+
+```bash
+sudo apt-get install -y build-essential git vim-nox wget curl
 ```
 
-## Installing the tools and components
+### :red_circle: Installing PostgreSQL
 
-Now we install some essential tools.
+Set the postgres repo for apt.
```

Comment:
maybe link here the source of this specific install instructions? To allow easier confirmation that it is the correct one and to make future updates easier?